docs/admin.rst

```diff
@@ -42,3 +42,52 @@ to your MPTTModelAdmin::
     class CustomMPTTModelAdmin(MPTTModelAdmin):
         mptt_indent_field = "some_node_field"
     # …
+
+
+``mptt.admin.DraggableMPTTAdmin``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 0.9
+
+This is a tree admin based on `FeinCMS <http://feincms.org/>`_ offering
+drag-drop functionality for moving nodes.
+
+.. warning::
+
+   Does not work well with big trees (more than a few hundred nodes, or trees
+   deeper than 10 levels). Patches implementing lazy-loading of deep trees
+   are very much appreciated.
+
+``DraggableMPTTAdmin.list_per_page`` is set to 2000 by default (which
+effectively disables pagination for most trees).
+
+Usage::
+
+    from django.contrib import admin
+    from mptt.admin import DraggableMPTTAdmin
+    from myproject.myapp.models import Node
+
+    admin.site.register(
+        Node,
+        DraggableMPTTAdmin,
+        list_display=('__str__', ...more fields if you feel like it...),
```

Comment:
Thanks — the magic mechanism here was really hard to explain. Therefore I removed the magic in  https://github.com/django-mptt/django-mptt/commit/690e48d46ef92ef07b3b760f3fd60f51f9ad7d6d